### DISCUSSION TITLE
Can't find my school in the school list or location

### DISCUSSION THREAD

**Discussion by AkinaLTS** [community/community#45032] [GitHub Education]
My school isn't registered on GitHub, and I tried three times to apply but couldn't find my school.  The information for each rejection is "Please provide the name of your school".  My school can't even be found on the map provided on the second page of the application student pack process.  But I'm really a high school freshman from China.
![IMG](https://user-images.githubusercontent.com/108107785/213896272-5a8e5e53-da06-4f0d-b39d-6dc142bde6ba.jpeg)

**Comment by jge162** [comment]
Are u using your school email and posting pictures of your transcripts. If not will be hard to prove. Also maybe get translated documents.

**Reply by AkinaLTS** [reply]
But the main problem is that I can't find my school, so that the school address on the application interface is inaccurate or even wrong.  And my school doesn't provide an educational email address.
